src/AccessControlHierarchyBase.php

```diff
@@ -212,16 +212,23 @@ public function disallowedOptions($field) {
     return array_keys($options);
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function getApplicableFields($entity_type, $bundle) {
```

Comment:
actually, this means there is no API change now, because of the base implementation 🎉 
so we don't need the change record either!

Reply:
I still think we should publish it, because the base implementation doesn't actually do anything. Anyone who wrote a plugin will need to implement that method.

Reply:
ok, it just needs updating then, as the change record says something like 'there is no base implementation' or similar
i reckon this is good to merge (Assuming its passing on d.o)